Terraform/cloudwatch-with-sns/boto/subscribe.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+
+import boto3
+import botocore
+import sys
+
+
+def main():
+    topic_arn = sys.argv[1]
+    access_key = sys.argv[2]
+    secret_key = sys.argv[3]
+    symphony_ip = sys.argv[4]
+
+   # print ("Disabling warning for Insecure connection")
```

Comment:
what is this comment for?

Reply:
Probably in some environment that "disable warning" was necessary.. prefer to keep that option if someone will need it.